dashboards/ansible-tower/ansible-tower.json

```diff
@@ -0,0 +1,801 @@
+          },
+          {
+            "title": "Memory usage (%)",
+            "layout": {
+              "column": 5,
+              "row": 1,
+              "width": 4,
+              "height": 3
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.line"
+            },
+            "rawConfiguration": {
+              "config": {
+                "bisect": {
+                  "position": null
+                }
+              },
+              "legend": {
+                "enabled": false
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": "3510613",
+                  "events": {
+                    "agent_ids": [
+                      "8263907339200612017"
+                    ]
+                  },
+                  "query": "SELECT average(memoryUsedPercent) AS 'Memory used %' FROM SystemSample WHERE `entityGuid` = 'MzUxMDYxM3xJTkZSQXxOQXw4MjYzOTA3MzM5MjAwNjEyMDE3' TIMESERIES auto"
+                }
+              ],
+              "yAxisLeft": {
+                "max": 100,
+                "min": 0,
+                "zero": null
+              }
+            }
+          },
+          {
+            "title": "Storage usage (%)",
+            "layout": {
+              "column": 9,
+              "row": 1,
+              "width": 4,
+              "height": 3
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.line"
+            },
+            "rawConfiguration": {
+              "config": {
+                "bisect": {
+                  "position": null
+                }
+              },
+              "legend": {
+                "enabled": false
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": "3510613",
+                  "events": {
+                    "agent_ids": [
+                      "8263907339200612017"
+                    ]
+                  },
+                  "query": "SELECT average(diskUsedPercent) as 'Storage used %' FROM StorageSample WHERE `entityGuid` = 'MzUxMDYxM3xJTkZSQXxOQXw4MjYzOTA3MzM5MjAwNjEyMDE3' TIMESERIES auto"
+                }
+              ],
+              "yAxisLeft": {
+                "max": 100,
+                "min": 0,
+                "zero": null
+              }
+            }
+          },
+          {
+            "title": "Network traffic",
+            "layout": {
+              "column": 1,
+              "row": 4,
+              "width": 8,
+              "height": 3
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.area"
+            },
+            "rawConfiguration": {
+              "actions": {
+                "point": {
+                  "logs": {
+                    "enabled": true,
+                    "entityGuid": "MzUxMDYxM3xJTkZSQXxOQXw4MjYzOTA3MzM5MjAwNjEyMDE3"
```

Comment:
This `entityGuid` is hard-coded. If we're unable to display logs without a specific entity, perhaps we should remove this widget.

Reply:
done.. removed it and updated the dashboard json